.github/workflows/compiler_sanitizers.yml

```diff
@@ -121,7 +121,7 @@ jobs:
     - name: Test
       run: |
         # These tests are slow, so only run tests in files that do "import threading" to make them count
-        TSAN_OPTIONS=allocator_may_return_null=1:halt_on_error=1 \
+        TSAN_OPTIONS="allocator_may_return_null=1:suppressions=tools\ci\tsan_suppressions.txt" \
```

Comment:
maybe try using an absolute path on the runner?

Reply:
I'll try that, assuming the absolute path is relative stable

Reply:
You can use an environment variable to get the base directory: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

Reply:
Ok, both GITHUB_WORKSPACE and RUNNER_WORKSPACE seem to point to the right location. Picking the first option.